apps/sms/index.html

```diff
@@ -608,13 +608,15 @@ <h1 class="message-subject">
 
     <div id="messages-group-header-tmpl" class="hide">
       <!--
-      <bdi>${name}</bdi><bdi dir="ltr"> (+${participantCount})</bdi>
+      <span class="group-header-title">
+        <bdi class="ellipsis-dir-fix">${name}</bdi><bdi dir="ltr"> (+${participantCount})</bdi>
+      </span>
       -->
     </div>
 
     <div id="messages-header-tmpl" class="hide">
       <!--
-      <bdi>${name}</bdi>
+      <bdi class="ellipsis-dir-fix">${name}</bdi>
```

Comment:
Maybe we can remove the -moz-fit-content and set auto to let text-align center to handle alignment if we apply the hack, because the original title container didn't expect any block element inside the title. Another idea is we apply display flex and justify-content center for all the cases, wdyt?